Frontend/src/ui/Competition.jsx

```diff
@@ -129,15 +129,17 @@ export default function Competition({ children }) {
                   </span>
                 ))}
               </div>
-              <div>
-                <span className={styles.eventHeader}>Main Event:</span>
-                <span className={styles.event}>
-                  <CubingIcon
-                    event={competitionInfo.main_event_id}
-                    selected={true}
-                  />
-                </span>
-              </div>
+              {competitionInfo.main_event_id && (
+                <div>
+                  <span className={styles.eventHeader}>Main Event:</span>
+                  <span className={styles.event}>
+                    <CubingIcon
+                      event={competitionInfo.main_event_id}
+                      selected={true}
+                    />
```

Comment:
We really need a 'non-selectable' version of this.
```suggestion
                    <CubingIcon
                      event={competitionInfo.main_event_id}
                      selected
                    />
```

Reply:
It isn't actually selectable by default in my code. That the monolith overwriting my CSS